app/config/config_dev.yml

```diff
@@ -3,14 +3,15 @@
 
 imports:
     - { resource: "config.yml" }
+    - { resource: "parameters_dev.yml" }
```

Comment:
Can't we just change a data of parameters.yml between `composer install` and `sylius:install`?